dm/tests/upstream_switch_mariadb/docker-compose.yml

```
@@ -0,0 +1,80 @@
+version: "3.7"
+
+networks:
+  db-networks:
+    ipam: # for static IP
+      config:
+        - subnet: "172.28.128.0/24"
+
+# re-usable extension fields, https://docs.docker.com/compose/compose-file/#extension-fields.
+x-node: &default-node
+  privileged: true
+  restart: always
+  environment:
+    MYSQL_ROOT_PASSWORD: "123456"
+
+services:
+  mysql57_master:
```

Can add a comment to explain this naming.
```suggestion
  # In Mariadb case, mysql57 represents mariadb binlog test; mysql8 represents mariadb gtid test. It has nothing to do with the version
  mysql57_master:
```